_posts/en/newsletters/2021-04-14-newsletter.md

```diff
@@ -0,0 +1,213 @@
+  interested in activation.  PRs were opened for two variants of it:
+  [PR#21377][Bitcoin Core #21377], using a variation on [BIP9][], and
+  [PR#21392][Bitcoin Core #21392], using a modification that became part
+  of [BIP8][].  The main technical difference between these PRs is how
+  their start and stop points were specified.  PR#21377 uses Median Time
+  Past (MTP)---the median time declared in the headers of the current
+  and previous 11 blocks.  PR#21392 uses the *height* of the current
+  block---the number of blocks between it and the *genesis block*, block
+  0.
+
+    MTP is normally roughly consistent between Bitcoin's main network
+    (mainnet) and its various testnet works, such as testnet, the default
+    [signet][topic signet], and various independent signets.  This
+    allows multiple networks to share a single set of activation
+    parameters even if they have vastly different block heights,
+    minimizing the work of keeping those networks' users in sync with
+    mainnet's consensus changes.
+
+    Unfortunately, MTP can be easily manipulated in small ways by a
+    small number of miners and in large ways by a majority of hash rate.
+    It can also revert to an earlier time even by accident during a
+    block chain reorganization (reorg).  By comparison, heights can only
+    decrease in extraordinary reorgs,[^height-decreasing] likely
+    requiring the obvious and prolonged manipulation by a majority of
+    miners.  That generally allows reviewers to make the simplifying
+    assumption that height will only ever increase, making it easier to
+    analyze height-based activation mechanisms than MTP mechanisms.
+
+    These tradeoffs between the two proposals, among other concerns,
+    created an impasse that some developers thought was preventing either
+    PR from receiving additional review and, ultimately, getting one of
+    them merged into Bitcoin Core.  That impasse was resolved to the
+    satisfaction of some participants in the activation discussion when
+    the authors of the two PRs agreed to a compromise:
+
+    1. To use MTP for the earliest time when nodes begin counting blocks
+       signaling for the soft fork.  This is identical to the way
+       [BIP9][] versionbits and [BIP148][] UASF started counting blocks
+       for the soft forks they helped activate.
+
+    2. To use MTP for the *earliest* time when nodes stop counting block
+       signaling for the soft fork, with counting continuing until the
+       end of the 2,016-block retarget period that reached the stop time.
+       This is a change from BIP9 and BIP148 where MTP was used for the
+       *latest* time when counting stopped.  The difference removes the
+       ability for an activation attempt to go directly from *started*
+       to *failed*, simplifying analysis and guaranteeing that miners
+       will get at least one retarget period to activate taproot.
```

Comment:
Perhaps "miners will get at least one retarget period to activate taproot" -> "there will be at least one retarget period when miners can signal for activation" ?